### ISSUE TITLE
https://wanderinginn.com/table-of-contents/ broken with recent update to site

### ISSUE THREAD

**Issue by boomboxnation** [dteviot/WebToEpub#984]
https://wanderinginn.com very recently overhauled their TOC page. (kind of unfortunate as it's now only broken into Volumes rather than books. And the Volumes are HUGE.

I will start trying my hand on editing the current parser. But decided to set the dev on it as well if they have the cycles. Thanks dteviot!

Edit: Oh...I see. There is no parser for it. What is the Parser for sites without a dedicated Parser.js? This was working for wanderinginn previously. In Advanced the Parser is just blank.

**Comment by dteviot** [comment]
@boomboxnation 
> There is no parser for it. What is the Parser for sites without a dedicated Parser.js?

There are some common site themes, e.g. Madara, "Basic" WordPress, WebToEpub will look for these and select the appropriate parser if the site looks like that.  Otherwise, it falls back to DefaultParser.

WebToEpub was selecting the WordPress parser for the site, but that's not working for finding the Table of Contents.

Anyway, I've added a new, minimal  parser for the site.  Feel free to expand.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with: 
* https://wanderinginn.com/table-of-contents/ and chapter 1
For my notes: 39 minutes work
